pkgs/misc/emulators/fceux/default.nix

```diff
@@ -1,11 +1,13 @@
-{stdenv, fetchurl, scons, zlib, SDL, lua5_1, pkgconfig}:
+{stdenv, fetchFromGitHub, scons, zlib, SDL, lua5_1, pkgconfig}:
 
 stdenv.mkDerivation {
-  name = "fceux-2.2.3";
+  name = "fceux-2.2.3-20200129";
```

Comment:
Until the next stable release we should use the following scheme ([source](https://nixos.org/nixpkgs/manual/#sec-package-naming)):
```nix
pname = "fceux-unstable";
version = "2020-01-29";
```

Reply:
Applied.